feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt

```diff
@@ -35,6 +67,140 @@ class SessionSurveyFragment : DaggerFragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         sessionSurveyFragmentArgs = SessionSurveyFragmentArgs.fromBundle(arguments ?: Bundle())
+
+        progressTimeLatch = ProgressTimeLatch { showProgress ->
+            binding.progressBar.isVisible = showProgress
+        }
+
+        sessionSurveyStore.loadingState.changed(viewLifecycleOwner) {
+            progressTimeLatch.loading = it.isLoading
+        }
+
+        sessionSurveyStore.sessionFeedback.changed(viewLifecycleOwner) { sessionFeedback ->
+            Timber.debug { sessionFeedback.toString() }
+            // TODO: save sessionFeedback state to cacheDB
+        }
+
+        val lang = defaultLang()
+        binding.sessionTitle.text = sessionSurveyFragmentArgs.session.title.getByLang(lang)
+        // TODO: set speackers icon
+
+        binding.submitButton.setOnClickListener {
+            val sessionFeedback =
+                sessionSurveyStore.sessionFeedback.requireValue().copy(submitted = true)
+            if (sessionFeedback.fillouted) {
+                // TODO: show confirm dialog
+                sessionSurveyActionCreator.submit(
+                    sessionSurveyFragmentArgs.session,
+                    sessionFeedback
+                )
+            } else {
+                // TODO: show snackbar no input item message
+            }
+        }
+
+        setupSessionSurveyPager()
+        sessionSurveyActionCreator.load(sessionSurveyFragmentArgs.session.id)
+    }
+
+    private fun setupSessionSurveyPager() {
+        val surveyItems = resources.getStringArray(R.array.survey_items)
+        binding.sessionSurveyViewPager.adapter = object : PagerAdapter() {
+            override fun instantiateItem(container: ViewGroup, position: Int): Any {
+                val itemBindng = DataBindingUtil.inflate<ItemSessionSurveyBinding>(
+                    LayoutInflater.from(context),
+                    R.layout.item_session_survey,
+                    container,
+                    true
+                )
+                itemBindng.title.text = surveyItems[position]
```

Comment:
Since it is important to be able to do it, even after merging it is OK, but making the following classes below "ui.widget" seems to be easy.

```
class SurveryItem(val title: Int, ...)
```